### ISSUE TITLE
open sources crash

### ISSUE THREAD

**Issue by oan22** [Bionus/imgbrd-grabber#1136]
### What steps will reproduce the problem?

open "sources"

### What is the expected behavior? What do you get instead?

see "sources"

### How often does this problem occur? On which sources?

all day, every day
gelbooru
danbooru
safebooru
atf
rule34.xxx

### What version of the program are you using? On what operating system?

5.5.2
Win7

**Comment by Bionus** [comment]
1. Did you add any source after installing the program?
2. Did you change any source's settings?
3. Can you share your `C:\Users\%USERNAME%\AppData\Local\Bionus\Grabber\main.log` file? (after a crash)
4. As well as your `C:\Users\%USERNAME%\AppData\Local\Bionus\Grabber\sites` folder?
5. Did you install 5.5.2 fresh or did you upgrade from a previous version?

**Comment by oan22** [comment]
I only added / modified one source (atf) in the version that preceded 5.5.2, once I upgraded I started crashing. Perhaps there was a dupe or something that caused conflicts.

I fixed the issue by just copying the sources from a default generated .cfg and placed them in my old .cfg. Here are the files after a crash anyways.

[main.log](https://github.com/Bionus/imgbrd-grabber/files/1448600/main.log)
[sites.zip](https://github.com/Bionus/imgbrd-grabber/files/1448605/sites.zip)
